ironfish/src/serde/PartialHeaderSerde.ts

```diff
@@ -14,8 +13,7 @@ export default class PartialBlockHeaderSerde {
     bw.writeHash(header.previousBlockHash)
     bw.writeHash(header.noteCommitment)
     bw.writeHash(header.transactionCommitment)
-    // TODO: Change to little-endian for consistency, since other non-bigint numbers are serialized as little-endian.
```

Comment:
Want to leave this comment until it's updated? Or do you think it should stay as BE

Reply:
I think it should stay as BE. It seemed to me like the target is not exactly a BigInt. It's more of a target hash value that the has of the block should be less than. For that reason I thought it made sense to serialize it in BE so that the bytes in the serialized format would be largest to smallest left to right to mirror a potential hash value. I don't think it makes too much of a difference but it did kind of make sense to me that it was BE

Reply:
Agreed that we will change the network and database serializations to be BE as well for consistency